src/lib/content/posts/post-card-image-test.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Note for @zspak98: The markdown files are test posts to show concepts, but will be deleted before merging.